README.md

```diff
@@ -15,27 +15,33 @@
 </a>
 
 # Generative data preparation
-This software package is designed for preparing data that can be used to train generative models. It offers an efficient way to convert input text files into tokenized sequences that are packed into a fixed sequence length. The resulting output directory can be directly used for training with SambaStudio. This package features many styles of packing text of any length into tokenized sequences, compressed HDF5 file outputs, efficient multiprocessing, shuffling any sized dataset, splitting your data into train/dev/test, and specifying what tokens are attended to during training.
+
+The [`pipeline.py`](https://github.com/sambanova/generative_data_prep/blob/main/generative_data_prep/data_prep/pipeline.py) script streamlines data preparation for machine learning model training. It takes a single input file, shuffles and splits it into train/dev/test files, and tokenizes, sequences, and converts them to HDF5 format using
+the utilities in [`data_prep.py`](https://github.com/sambanova/generative_data_prep/blob/main/generative_data_prep/data_prep/data_prep.py). The output directory contains multiple split HDF5 files that are needed to run data parallel training. This output directory will be directly used as a training dataset in SambaStudio. While this package features simple flows that work out of the box, it also supports more customization allowing for many styles of packing text of any length into tokenized sequences.
+
+</br>
 
 ## Table of contents
-- [Contributing](#Contributing)
-- [Installation](#installation)
 - [Requirements](#requirements)
-- [Introduction](#introduction)
-- [Input format](#input-format)
-- [End to end data preparation](#end-to-end-data-preparation)
-    - [Input Flags](#flags)
-- [Tokenizing one file](#tokenizing-one-file)
-    - [Input Flags](#tokenize-one-file-flags)
-- [Running tests](#running-tests)
-- [Example use cases](#example-use-cases)
+- [Installation](#installation)
+- [Get Started](#end-to-end-data-preparation)
+- [Input](#input)
+- [Output](#output)
+- [Flags](#flags)
+- [Examples](#example-use-cases)
     - [Pretraining](#pretraining)
-    - [Generative tuning](#generative-tuning)
+    - [Fine-tuning](#fine-tuning)
```

Comment:
Across the PR I switched "Generative Tuning" to "Fine-Tuning". My rationale here is that the term "Fine-tuning" is a more ubiquitous term in the GenAI space currently compared to Generative Tuning, so I felt like it would make sense to default to the more common term and add "Generative Tuning" in parentheses as opposed the converse. I think this may have some chain effects throughout the repo though with variable names and/or comments so I understand if this is a tough change to agree with. Eager to hear your thoughts though!